docs/user/accumulo/install.rst

```diff
@@ -82,7 +82,7 @@ Namespace Install
 ^^^^^^^^^^^^^^^^^
 
 Copying the runtime JAR to each tablet server as above will work, but in
-Accumulo 1.6+, we can leverage namespaces to isolate the GeoMesa classpath
+Accumulo 1.7+, we can leverage namespaces to isolate the GeoMesa classpath
```

Comment:
hmm, should we leave this as 1.6? we don't technically support it, but it will still work with namespaces... or maybe just remove the version reference altogether: `but in Accumulo we can leverage`